README.md

```diff
@@ -1 +1,62 @@
 # Manifest Schema
+
+This project contains publicly available json schemas that define slacks `manifest.json` file. JSON schemas can be used for a number of tasks but the purpose of this specific schema is to provide developers validation and prediction capabilities while editing `manifest.json` files in IDEs.
+
+**Guide Outline**:
+
+- [Manifest Schema](#manifest-schema)
+  - [Usage](#usage)
+  - [Project Structure](#project-structure)
+    - [`manifest.schema.json`](#manifestschemajson)
+    - [`/schemas`](#schemas)
+    - [`/tests`](#tests)
+    - [`/scripts`](#scripts)
+  - [Resources](#resources)
+
+---
+
+## Usage
+
+JSON schemas can be used for various tasks, the goal of this project is to provide a JSON schema that defines slacks `manifest.json` and provides validation and IntelliSense capabilities to developers editing `manifest.json` files in IDEs that support these features.
```

Comment:
Think we can remove this line as it is a repeat of the introductory sentence.

Reply:
True this